rustls/src/verify.rs

```diff
@@ -468,40 +468,22 @@ impl AllowAnyAuthenticatedClient {
     /// Construct a new `AllowAnyAuthenticatedClient`.
     ///
     /// `roots` is the list of trust anchors to use for certificate validation.
-    pub fn new(roots: RootCertStore) -> Self {
+    /// `crls` are an iterable of owned certificate revocation lists (CRLs) to use for
+    /// client certificate validation.
+    pub(crate) fn new(
+        roots: RootCertStore,
+        crls: impl IntoIterator<Item = webpki::OwnedCertRevocationList>,
```

Comment:
I changed things so that the builder does the work of converting from `UnparsedCertRevocationList` to `webpki::OwnedCertRevocationList` so that errors can be surfaced at the time of calling `with_crls(...)`. As a result the verifier side is changed to work with the owned representation and is infallible to construct.

Reply:
Good point, fixed.